### ISSUE TITLE
Enable admins (or the public) to tell who (admin or user) made a deposit to ScholarSphere using RMD's deposit workflow

### ISSUE THREAD

**Issue by anaelizabethenriquez** [psu-libraries/researcher-metadata#765]
It would be nice to have more breadcrumbs to tell where deposits in ScholarSphere came from. It may be that they are already there (either in ScholarSphere or in RMD) and we just need to know what to look for. 

It would be especially helpful to know when a deposit was made by an admin or proxy acting on behalf of the researcher who is listed as the depositor in ScholarSphere. And it would be great to know who the proxy or admin was, if that is possible.

This feature would help Scholarly Communications and Copyright and the ScholarSphere team when we are troubleshooting/curating deposits like this one, where the file uploaded isn't actually the kind of thing we want in ScholarSphere. The ScholarSphere curation team would like to be able to follow up and get the right file, but the user listed as the depositor may not be expecting an email from them (if the "depositor" didn't actually make the deposit).

**Comment by anaelizabethenriquez** [comment]
This would be useful related to the new curation workflow for ScholarSphere, because it would set us up to be able to have certain deposits from RMD skip the curation workflow (e.g. if an RMD admin already reviewed/curated the deposit).

**Comment by ajkiessl** [comment]
- [ ] We can easily differentiate between the Activity Insight OA Workflow uploads to Scholarsphere and the ScholarSphere Deposit Workflow with two differently named API keys.   We should rename the current ScholarSphere Deposit Workflow API key and add an API key for the Activity Insight OA Workflow.
- [ ] We would like to capture who the actual depositor is (author, proxy, or admin).  This will likely require ScholarSphere to allow another metadata field accepted by the ScholarSphere deposit API endpoint.  We should display in the ScholarSphere breadcrumbs who this person is and their role.

**Comment by anaelizabethenriquez** [comment]
@bdezray

**Comment by anaelizabethenriquez** [comment]
@ajkiessl Is there anything we need to do on the RMD side to ensure RMD is ready to share this info with ScholarSphere once the ScholarSphere changes are in place?

**Comment by anaelizabethenriquez** [comment]
@ajkiessl Do we need to add an issue about this to the ScholarSphere backlog?

@bdezray

**Comment by ajkiessl** [comment]
@anaelizabethenriquez Yeah, here is the issue: https://github.com/psu-libraries/scholarsphere/issues/1422

**Comment by anaelizabethenriquez** [comment]
Seems like this specific feature will be less relevant once users are depositing directly within ScholarSphere, so I am deprioritizing this.
